### PR TITLE
packet drop issue resolution

### PR DESCRIPTION
**Pull request by amarsri28**
This is preliminary patch .

### CODE REVIEW

File: core/modules/measure.cc [hunk 1]
```diff
@@ -132,21 +129,20 @@ void Measure::ProcessBatch(Context *ctx, bess::PacketBatch *batch) {
...
+      rtt_hist_.AtomicInsert(diff);
       if (rand_.GetRealNonzero() <= jitter_sample_prob_) {
         if (unlikely(!last_rtt_ns_)) {
           last_rtt_ns_ = diff;
           continue;
         }
-        uint64_t jitter = absdiff(diff, last_rtt_ns_);
-        jitter_hist_.Insert(jitter);
+        int64_t jitter = diff - last_rtt_ns_;
+        jitter = jitter * ((jitter>0) - (jitter<0));
```
Comment (gab-arrobo) [inline core/modules/measure.cc:139]: Why not just use the `abs`
```suggestion
        int64_t jitter = abs(diff - last_rtt_ns_);
```
Of course, the above would need to include `<cstdlib>` if needed

Comment (gab-arrobo) [inline core/modules/measure.cc:139]: You need to apply `clang-format` to address the GHA issue

**Review by gab-arrobo — DISMISSED** [review]
+1

File: core/modules/measure.cc (hunk 1 above)
Comment (gab-arrobo) [inline core/modules/measure.cc:139]: Moreover, why do you have to change `jitter` to `int64_t` given that the argument for `jitter_hist_.AtomicInsert()` is `uint64_t?

**Review by gab-arrobo — COMMENTED** [review]
Please address the previous comments. Thanks!

**Review by sureshmarikkannu — APPROVED** [review]
looks good

**Comment by gab-arrobo** [comment]
@amarsri28, any update about this PR?

**Pull request closed without merging**